core/dbt/artifacts/resources/v1/semantic_model.py

```diff
@@ -127,25 +133,11 @@ def is_linkable_entity_type(self) -> bool:
 
 
 # ====================================
-# Measure objects
+# Measure object
 # Measure protocols: https://github.com/dbt-labs/dbt-semantic-interfaces/blob/main/dbt_semantic_interfaces/protocols/measure.py
 # ====================================
 
 
-@dataclass
-class MeasureAggregationParameters(dbtClassMixin):
-    percentile: Optional[float] = None
-    use_discrete_percentile: bool = False
-    use_approximate_percentile: bool = False
-
-
-@dataclass
-class NonAdditiveDimension(dbtClassMixin):
-    name: str
-    window_choice: AggregationType
-    window_groupings: List[str]
-
-
```

Comment:
In `__init__.py` we are importing these classes from this `semantic_model`. That'll technically still work since we're also importing them into this file now. However, we should probably update `__init__.py` to import these two classes from `semantic_layer_components` now.